code/datums/craft/recipes/weapon.dm

```diff
@@ -59,7 +59,48 @@
 		list(/obj/item/stack/material/wood, 5, time = 25)
 	)
 
+/datum/craft_recipe/weapon/teleportation_spear
+	name = "telespear"
+	result = /obj/item/weapon/tele_spear
+	steps = list(
+		list(/obj/item/stack/rods, 2, time = 30),
+		list(/obj/item/weapon/stock_parts/subspace/crystal, 1),
+		list(/obj/item/device/assembly/igniter, 1),
+		list(/obj/item/stack/cable_coil, 10, time = 10)
+	)
 
+/datum/craft_recipe/weapon/improvised_maul
+	name = "robust maul"
+	result = /obj/item/weapon/melee/toolbox_maul
+	steps = list(
+		list(/obj/item/weapon/mop, 1, time = 30),
+		list(/obj/item/stack/cable_coil, 10, time = 10)
+	)
 
+/datum/craft_recipe/weapon/nailed_bat
+	name = "nailed bat"
+	result = /obj/item/weapon/melee/nailstick
+	steps = list(
+		list(/obj/item/stack/material/wood, 10, time = 30),
+		list(/obj/item/stack/rods, 3, time = 50)
+	)
 
+/datum/craft_recipe/weapon/handmade_shield
```

Comment:
A personal request, optional
I'd really like to see a shield being possible to create by combining a dinner tray and two belts
could you add that recipe too?

Reply:
Sure